openapi-cli/src/test/resources/ballerina-to-openapi/json_payload_service.bal

```diff
@@ -0,0 +1,18 @@
+ import ballerina/http;
+ import ballerina/log;
+
+ listener http:Listener helloEp = new (9090);
+
+ service /payloadV on helloEp {
+
+     //resource function post hi(http:Caller caller, http:Request request, int id) {
+     //
+     //}
```

Comment:
Are we going to uncomment these later ?

Reply:
we can remove this comments , I will remove